test/datetime/create.test.js

```diff
@@ -695,7 +695,7 @@ test("DateTime.fromObject overrides the locale string with explicit settings", (
     }
   );
 
-  expect(res.locale).toBe("be");
+  expect(res.locale).toBe("be-u-ca-coptic-nu-mong");
   expect(res.outputCalendar).toBe("islamic");
   expect(res.numberingSystem).toBe("thai");
 });
```

Comment:
Can you also add a test here where you set the `locale` to something with a calendar *and* an explicit `outputCalendar` and show that it doesn't blow up (I don't care which calendar "wins", but the test should document the result, i.e. which calendar comes out of the resolved options)

Reply:
there's a test for that right below this one!

Reply:
oh, ha, you're right. My bad